.github/workflows/publish-oss-for-cloud.yml

```diff
@@ -90,37 +91,55 @@ jobs:
         with:
           branch_version_tag: ${{ needs.generate-tags.outputs.dev_tag }}
 
+      - name: Publish Dev Jars
+        env:
+          CLOUDREPO_USER: ${{ secrets.CLOUDREPO_USER }}
+          CLOUDREPO_PASSWORD: ${{ secrets.CLOUDREPO_PASSWORD }}
+        run: VERSION=${{ needs.generate-tags.outputs.dev_tag }} SUB_BUILD=PLATFORM ./gradlew publish
+        shell: bash
+
+      - name: Publish Master Jars
```

Comment:
remind me again why we are publishing master jars with this if condition?

Reply:
This was in the tech spec https://docs.google.com/document/d/1Os-GMcDwJuqVNKI8yUmSbOevYt92N1W3BWY-Dj8DqdY/edit#heading=h.pyzrac8d5gsg where commits from the master branch would create different tags/jar versions from non master commits.  I think we can actually drop this feature though because I don't actually see how we'd use these tags.  I think it'd be easier to just assume the tag will always be `dev-<short_sha>`

Reply:
Thanks Peter. I do think it's useful though agree this is an incremental. I think it's worth adding a comment here explaining why. also think it's fine to remove if it seems confusing.

Reply:
yeah, I think we can just leave it as is for now, but if publishing master jars seems to noticeably slow down this workflow then we can probably remove this step and comment on why